x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/service_groups_list.tsx

```diff
@@ -29,7 +30,7 @@ export function ServiceGroupsListItems({ items, servicesCounts }: Props) {
       {items.map((item) => (
         <ServiceGroupsCard
```

Comment:
You probably need a `key` property in here, don't you? 

```suggestion
        <ServiceGroupsCard key={item.id}
```